net-misc/sstp-client/sstp-client-1.0.10-r1.ebuild

```diff
@@ -9,16 +9,16 @@ DESCRIPTION="A client implementation of Secure Socket Tunneling Protocol (SSTP)"
 HOMEPAGE="http://sstp-client.sourceforge.net/"
 SRC_URI="mirror://sourceforge/sstp-client/${P}.tar.gz"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+-with-openssl-exception"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 x86"
 IUSE="static"
 
 RDEPEND=">=dev-libs/libevent-2.0.10
-	dev-libs/openssl:0
-	net-dialup/ppp:="
+		dev-libs/openssl:0
```

Comment:
Whatever nonsense you have read, it makes no sense. Gentoo indents with *1 tab* for one level of indentation, not some magical recalculation between tabs and spaces.